### PR TITLE
display update

### PR DESCRIPTION
**Pull request by Mxdcq**

### CODE REVIEW

File: app/css/style.css [hunk 1]
```diff
@@ -7,6 +7,20 @@ html, body, iframe {
 
 md-card[md-ink-ripple] {
   position: relative;
+  width:100%;
+  /*overflow-y: auto;*/
+}
+
+p {
```
Comment (antti-TIEA4) [inline app/css/style.css]: Maybe it is not a good idea to change every `<p>`
Could you make this specific to some class?

Comment (Mxdcq) [inline app/css/style.css]: Sure

**Comment by Mxdcq** [comment]
Updated.

File: app/css/style.css [hunk 2]
```diff
@@ -7,7 +7,7 @@ html, body, iframe {
 
 md-card[md-ink-ripple] {
   position: relative;
-  width:100%;
+  /*width:100%;*/
   /*overflow-y: auto;*/
```
Comment (antti-TIEA4) [inline app/css/style.css]: Hmm... if both of these are now commented out, do we need either of them? Could be just deleted altogether.

**Comment by Mxdcq** [comment]
do as you said. comments deleted.

**Review by ghost — APPROVED** [review]

**Comment by Mxdcq** [comment]
I can merge it by myself?

**Comment by antti-TIEA4** [comment]
@Mxdcq In the future, when you push commits, it will help if you describe what you did in each change.
For example "Delete comments" is much more helpful than "little change"
This will help the people who look at our code base to see what has been changed and where.
I squashed and gave a new description to the commit to make it look better.
:+1:

**Comment by Mxdcq** [comment]
Thanks for your advice.

**Comment by antti-TIEA4** [comment]
@Mxdcq 
Here's how the changes look like: https://github.com/APT-project/upcoming/commits/master

In fact the commit you made 6 days ago:

> **replace key filed with my google api key**

is a good example of what a good commit message title should be like.
It tells briefly what has happened. :+1:

Here's a good article if you want to read more about writing commit messages: http://chris.beams.io/posts/git-commit/

**Pull request merged**